imports/server/fixture.js

```diff
@@ -42,11 +42,13 @@ Meteor.methods({
       }, {
         $set: {
           hunt: huntId,
-          tags: puzzle.tags,
           title: puzzle.title,
           url: puzzle.url,
           answer: puzzle.answer,
         },
+        $addToSet: {
```

Comment:
Not that this is wrong, but FWIW I consider it a bug in the fixture if any of them have duplicate tag ids

Reply:
Yeah, I agree, but I felt like being exhaustive.